test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -52,6 +52,30 @@ def wrapper(self):
         return func(self)
     return wrapper
 
+# skips tests for all versions below min_opset_version.
```

Comment:
Nits: move functions to https://github.com/pytorch/pytorch/blob/master/test/onnx/test_pytorch_common.py